entry/src/main/resources/base/element/string.json

```diff
@@ -0,0 +1,32 @@
+{
+  "string": [
+    {
+      "name": "entry_MainAbility",
```

Comment:
All the strings values must be meaningful and must be related to the library. Replace the generic/default strings or remove them if not required